style.css

```diff
@@ -499,6 +501,58 @@ footer {
   display: none;
 }
 
+@media screen and (max-width: 768px) {
+  .open-menu {
+    display: block;
+    top: 50px;
+    right: 20px;
+    color: #6070ff;
+    margin-right: 5px;
+    font-size: 20px;
+  }
+
+  .close-menu {
+    display: none;
+    position: absolute;
+    top: 20px;
+    right: 25px;
+    font-size: 1.5rem;
+    font-weight: 600;
+    z-index: 2;
+  }
+
+  .nav-links {
+    display: none;
+    position: fixed;
+    flex-direction: column;
+    height: 100vh;
+    top: 0;
+    right: 0;
+    left: 0;
+    z-index: 2;
+    background: #3c3a39;
+    opacity: 0.9;
+    padding: 60px 0 0 20px;
+    transition: 0.7s ease-in-out;
+  }
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly the the missing white background and the black line at the bottom of the menu container to match the figma design.

![hey](https://user-images.githubusercontent.com/59397159/167713866-e5d76088-32da-45cf-92d6-25c063684dfc.png)